app/views/matchers/_result.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if api_responses[:error] %>
+  Sorry, something screwed up!
+<% else %>
+  <div class="col-md-6">
+    <h3>Found <%= api_responses[:cfpb_complaint_count] %> CFPB complaints for <%= api_responses[:cfpb_random_complaint]['company'].titleize %></h3>
+
+    <p><%= api_responses[:cfpb_random_complaint]['complaint_what_happened'] %></p>
```

Comment:
An enhancement here would be to pull 'random' complaints that favored things with complaint text.